addons/account/models/account_move.py

```diff
@@ -2835,7 +2835,7 @@ def _set_price_and_tax_after_fpos(self):
         # adapt the price_unit to the new tax.
         # E.g. mapping a 10% price-included tax to a 20% price-included tax for a price_unit of 110 should preserve
         # 100 as balance but set 120 as price_unit.
-        if self.tax_ids and self.move_id.fiscal_position_id:
+        if self.tax_ids and self.move_id.fiscal_position_id and self.move_id.fiscal_position_id.tax_ids:
```

Comment:
replace `business_vals = self._get_fields_onchange_balance(balance=balance)` by `business_vals = self._get_fields_onchange_balance(balance=balance, price_subtotal=price_subtotal)` instead.

2.4 / 1.14 = 2.105263158 ~= 2.11 but 2.11 * 1.14 = 2.4054 ~= 2.41

Reply:
EDIT: Forget about this: this is a bug in the compute_all method with `handle_price_include`. I will fix it tomorrow